subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/suppliers/DynamicRevisionRemoteResolveWithMetadataSupplierIntegrationTest.groovy

```diff
@@ -1269,7 +1269,7 @@ group:projectB:2.2;release
         file << """import org.gradle.api.artifacts.ComponentMetadataSupplier
           import org.gradle.api.artifacts.ComponentMetadataSupplierDetails
           import org.gradle.api.artifacts.repositories.RepositoryResourceAccessor
-          import javax.inject.Inject
+          import javax.inject.Inject;
```

Comment:
:x: this import statement can be removed now

Reply:
Thanks for catching